package.json

```diff
@@ -102,7 +102,7 @@
     "eslint-plugin-simple-import-sort": "^7.0.0",
     "ethers": "^5.4.6",
     "graphql": "^15.5.0",
-    "graphql-request": "^3.4.0",
+    "graphql-request": "^4.2.0",
```

Comment:
I introduced this change to force the CI script to run the `postinstall` npm script which runs the `yarn graphql:generate` script which generates the graphql TS types in `src/state/data/generated.ts` that is needed to import the `EnsNamesQuery` type.